### DISCUSSION TITLE
Error codes documentation

### DISCUSSION THREAD

**Discussion by romainstraet** [supabase/supabase#3574] [Questions]
Hi,
I'm currently discovering superbase.
I'm looking for some documentation regarding the error codes message that supabase could return (e.g.: like [firebase](https://firebase.google.com/docs/reference/js/v8/firebase.auth.Auth#createuserwithemailandpassword)) but I can't find it. Any ideas?
Thanks for your help,
Romain

**Comment by zakaria-chahboun** [comment]
It's just the postgres error codes
